templates/observability-remote-write-proxy.yml

```diff
@@ -260,6 +291,44 @@ parameters:
     Required when OBSERVABILITY_REMOTE_WRITE_PROXY_TOKEN_VERIFICATION_ENABLED
     is "true"
 
+- name: OBSERVABILITY_REMOTE_WRITE_PROXY_TOKEN_VERIFICATION_SERVICE_SERVING_CA_ENABLED
+  description: >-
+    If set to true, the provided Service Serving CA certificate
+    specified in the file pointed by
+    OBSERVABILITY_REMOTE_WRITE_PROXY_TOKEN_VERIFICATION_SERVICE_SERVING_CA_FILEDIR/OBSERVABILITY_REMOTE_WRITE_PROXY_TOKEN_VERIFICATION_SERVICE_SERVING_CA_FILENAME
+    is appended to the system's certificate trusted store when token verification
+    is performed. See OBSERVABILITY_REMOTE_WRITE_PROXY_TOKEN_VERIFICATION_ENABLED
+    for details on token verification.
+    The Serivce Serving CA certificate has to be a X.509 certificate encoded
+    in PEM format.
+    Only takes effect when OBSERVABILITY_REMOTE_WRITE_PROXY_TOKEN_VERIFICATION_ENABLED
+    is set to true.
+  value: "false"
```

Comment:
should we default to `true`?

Reply:
No. `false` is enough

Reply:
Make sense.

In that case, I suggest we default to the KFM route https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/0d7ca274d5133af95a897140133c133b83e43dbe/Makefile#L995 to the deployed KFM route?

something like https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/0d7ca274d5133af95a897140133c133b83e43dbe/Makefile#L898 + the token verification endpoint

What do you think?

Reply:
This assumes that KFM has been deployed beforehand.
I prefer to keep that independent.